.github/workflows/cypress.yml

```
@@ -0,0 +1,161 @@
+    #     run: |
+    #       echo "=== Copying .env to Container ==="
+    #       docker cp .env talawa-api-1:${{ env.CONTAINER_PATH }}/.env
+    #       docker exec talawa-api-1 ls -la ${{ env.CONTAINER_PATH }}/.env
+
+      - name: Copying .env to Container
+        run: |
+          echo "=== Copying .env to Container ==="
+          # First, check container working directory and structure
+          echo "Container working directory and contents:"
+          docker exec talawa-api-1 /bin/bash -c 'pwd && ls -la'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)